ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -922,6 +847,28 @@ def test_ipa_migrate_stage_mode_with_invalid_cert(self):
         assert result.returncode == 1
         assert ERR_MSG in result.stderr_text
 
+    def test_ipa_migrate_subids(self, empty_log_file):
```

Comment:
The tests test_ipa_migrate_subids, test_ipa_hbac_rule_duplication, test_ipa_migrate_check_service_status, test_custom_idrange_is_migrated and test_automountlocation_is_migrated can be moved to the class TestIPAMigrationProdMode.
Since they are all using the same ipa-migrate options, no need to call the command on each test:
- prepare the data in prepare_ipa_server
- call ipa-migrate once in test_ipa_migrate_prod_mode
- in the next tests, only check the presence of expected data in the new server